pyscript.core/src/stdlib/pyscript/magic_js.py

```diff
@@ -10,6 +10,11 @@
 
 config = json.loads(globalThis.JSON.stringify(_config))
 
+if "MicroPython" in sys.version:
```

Comment:
I usually normalise to: `if "micropython" in sys.version.lower()` But I don't think it makes much difference.

Reply:
I believe they really want the MicroPython brand to be **MicroPython** but I don't mind applying that change ... alternatively I could use other try/catches or pass the type from the page but that is actually pretty convoluted due workers VS main VS config in PyScript VS Polyscript and so on ... I think this was the fastest and easy way around.

Reply:
Yeah... it's also the smallest possible change. We can refine later if there are problems / feature requests. Was just mentioning it as a sort of "out of interest..." point. :+1: